addons/radios/XEH_postInit.sqf

```diff
@@ -0,0 +1,11 @@
+#include "script_component.hpp"
+
+["16aa", "16AA_Key_Adjust_Headset", "Lower/Raise Radio Headset",
+{
+	if!([ACE_player] call FUNC(canLowerHeadset)) exitWith {false};
+
+	[ACE_player] call FUNC(lowerHeadset);
+	ACE_player;
```

Comment:
Return value of a key handler should not be an object. Rather return a boolean here. Probably `true` ?